frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'common/types/types';
+import { useState } from 'react';
+
+import styles from '../../uam/components/group-creation/group-creation.module.scss';
+
+type Props = {
+  styles: {
+    wrapperClass: string;
+    inputClass: string;
+  };
+};
+
+const Checkbox: FC<Props> = ({ styles }) => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const handleChange = (): void => {
+    setIsChecked((prev) => !prev);
+  };
+
+  return (
+    <div className={styles.wrapperClass}>
+      <input
+        type="checkbox"
+        checked={isChecked}
+        onChange={handleChange}
+        className={isChecked ? styles.inputClass : ''}
```

Comment:
```suggestion
        className={isChecked && styles.inputClass}
```
Can be simplified to this